pipeline/coaddition.py

```diff
@@ -145,6 +171,8 @@ def _estimate_background(self, data):
 
         return bkg, sigma
 
+    # ======================================================================
+
```

Comment:
This is another case where I violate style guides.  I've seen style guides that say "don't put commented lines of dashes separating functions".  But, I find that doing this between functions, or between collections of functions, can really improve readability and help when I'm scanning through code quickly to realize that I've moved from one function to another.

Of course, I'm not consistent about it... and, because lots of people (well, three or four) edit this code, and I'm probably the only one who does this, it's even more inconsistent.  But, I did intend to leave them there.

Reply:
I actually tend to do this too myself, since I find it helpful when looking through the file, so I'm all for it.